src/main/java/rx/internal/operators/OperatorObserveOn.java

```diff
@@ -145,106 +150,126 @@ public void onCompleted() {
         @Override
         public void onError(final Throwable e) {
             if (isUnsubscribed() || finished) {
+                RxJavaPlugins.getInstance().getErrorHandler().handleError(e);
```

Comment:
More PRs to the same class ends up being merged much later and usually in the wrong order.